source/manual/get-started.html.md

```diff
@@ -369,6 +369,15 @@ to find out how to deploy infrastructure changes. The stackname is `govuk` and t
 
 See the [AWS IAM users documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html) for more information.
 
+Lastly, you will need to add the integration permissions provided in `user-reviewer`, for example:
```

Comment:
Tiny nit pick could we make user-reviewer a link to the repo?

Reply:
I've added a link and updated the link text to "GOV.UK User Reviewer" to keep it consistent with the existing documentation.